assets/src/edit-story/migration/migrations/v0018_conicToLinear.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function conicToLinear({ pages, ...rest }) {
+  return {
+    pages: pages.map(reducePage),
+    ...rest,
+  };
+}
+
+function reducePage({ elements, ...rest }) {
+  return {
+    elements: elements.map(updateElement),
+    ...rest,
+  };
+}
+
+function updateElement(element) {
+  const { backgroundColor } = element;
+  if (backgroundColor?.type !== 'conic') {
+    return element;
+  }
+  return {
+    ...element,
+    backgroundColor: {
+      ...backgroundColor,
+      type: 'linear',
```

Comment:
Conic gradients can have a `center`, linear gradients can't. However, there's currently no way to manipulate the center, so no conic gradients will have one. So this is just a convoluted way of saying, that we're good 🤝